src/main/java/seedu/guest/MainApp.java

```diff
@@ -54,14 +54,14 @@ public void init() throws Exception {
         AppParameters appParameters = AppParameters.parse(getParameters());
         config = initConfig(appParameters.getConfigPath());
 
-        UserPrefsStorage userPrefsStorage = new JsonUserPrefsStorage(config.getUserPrefsFilePath());
-        UserPrefs userPrefs = initPrefs(userPrefsStorage);
-        GuestBookStorage guestBookStorage = new JsonGuestBookStorage(userPrefs.getGuestBookFilePath());
-        storage = new StorageManager(guestBookStorage, userPrefsStorage);
+        GuestPrefsStorage guestPrefsStorage = new JsonGuestPrefsStorage(config.getGuestPrefsFilePath());
```

Comment:
Similar to the comment above at the import statements, should we revert back to the original class names and variables?